enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/WebSocketAPI.java

```diff
@@ -248,4 +277,59 @@ public WebSocketThrottleResponse processFramedata(RequestContext requestContext)
                 (Long) requestContext.getProperties().get(ThrottleConstants.HEADER_RETRY_AFTER));
         return webSocketThrottleResponse;
     }
+
+    private void populateRemoveAndProtectedHeaders(RequestContext requestContext) {
```

Comment:
shall we move this to the `Utils` class and use the same function in RestAPI and WebScoket API?